This patch is created by Oracle to:

    1. Add reference to dns-server(8s), SMF manual page.

    2. Add link to online ARM.

    3. Add a note regarding compile-time features.

This patch is Solaris specific and is not applicable to upstream.

--- bind-9.18.28/doc/man/named.8in
+++ bind-9.18.28/doc/man/named.8in
@@ -299,7 +299,20 @@ The default process\-id file.
 .UNINDENT
 .SH SEE ALSO
 .sp
-\X'tty: link https://datatracker.ietf.org/doc/html/rfc1033.html'\fI\%RFC 1033\fP\X'tty: link', \X'tty: link https://datatracker.ietf.org/doc/html/rfc1034.html'\fI\%RFC 1034\fP\X'tty: link', \X'tty: link https://datatracker.ietf.org/doc/html/rfc1035.html'\fI\%RFC 1035\fP\X'tty: link', \X'tty: link #std-iscman-named-checkconf'\fI\%named\-checkconf(8)\fP\X'tty: link', \X'tty: link #std-iscman-named-checkzone'\fI\%named\-checkzone(8)\fP\X'tty: link', \X'tty: link #std-iscman-rndc'\fI\%rndc(8)\fP\X'tty: link', \X'tty: link #std-iscman-named.conf'\fI\%named.conf(5)\fP\X'tty: link', BIND 9 Administrator Reference Manual.
+\X'tty: link https://datatracker.ietf.org/doc/html/rfc1033.html'\fI\%RFC 1033\fP\X'tty: link', \X'tty: link https://datatracker.ietf.org/doc/html/rfc1034.html'\fI\%RFC 1034\fP\X'tty: link', \X'tty: link https://datatracker.ietf.org/doc/html/rfc1035.html'\fI\%RFC 1035\fP\X'tty: link', \fBdns\-server(8s)\fP, \X'tty: link #std-iscman-named-checkconf'\fI\%named\-checkconf(8)\fP\X'tty: link', \X'tty: link #std-iscman-named-checkzone'\fI\%named\-checkzone(8)\fP\X'tty: link', \X'tty: link #std-iscman-rndc'\fI\%rndc(8)\fP\X'tty: link', \X'tty: link #std-iscman-named.conf'\fI\%named.conf(5)\fP\X'tty: link', BIND 9 Administrator Reference Manual.
+.PP
+The BIND 9 \fIAdministrator's Reference Manual\fR is available
+online at https://bind9.readthedocs.io/en/vCOMPONENT_VERSION/
+.PP
+.br
+.ps +1
+\fBNote\fR
+.ps -1
+.br
+Some compile-time features documented in the ARM are not available; to
+see a list of enabled features run '\fB/usr/sbin/named -V\fR'.
+.sp .5v
+.PP
 .SH AUTHOR
 Internet Systems Consortium
 .SH COPYRIGHT
